openvpn-install.sh

```diff
@@ -146,48 +124,38 @@ prefetch: yes' >> /etc/unbound/unbound.conf
 	directory: "/etc/unbound"
 	trust-anchor-file: trusted-key.key
 	root-hints: root.hints
-	interface: 10.8.0.1
-	access-control: 10.8.0.1/24 allow
 	port: 53
 	num-threads: 2
-	use-caps-for-id: yes
 	harden-glue: yes
-	hide-identity: yes
-	hide-version: yes
-	qname-minimisation: yes
-	prefetch: yes' > /etc/unbound/unbound.conf
+	qname-minimisation: yes' >/etc/unbound/unbound.conf
 		fi
+	fi
 
-		if [[ ! "$OS" =~ (fedora|centos|amzn) ]];then
-			# DNS Rebinding fix
-			echo "private-address: 10.0.0.0/8
-private-address: 172.16.0.0/12
-private-address: 192.168.0.0/16
-private-address: 169.254.0.0/16
-private-address: fd00::/8
-private-address: fe80::/10
-private-address: 127.0.0.0/8
-private-address: ::ffff:0:0/96" >> /etc/unbound/unbound.conf
-		fi
-	else # Unbound is already installed
-		echo 'include: /etc/unbound/openvpn.conf' >> /etc/unbound/unbound.conf
-
-		# Add Unbound 'server' for the OpenVPN subnet
-		echo 'server:
+	# Add Unbound 'server' for the OpenVPN subnet
+	mkdir -p /etc/unbound/unbound.conf.d
+	echo 'server:
 interface: 10.8.0.1
 access-control: 10.8.0.1/24 allow
 hide-identity: yes
 hide-version: yes
 use-caps-for-id: yes
-prefetch: yes
-private-address: 10.0.0.0/8
+prefetch: yes' >/etc/unbound/unbound.conf.d/openvpn.conf
+
+	if [[ ! "$OS" =~ (fedora|centos|amzn) ]];then
+		# DNS Rebinding fix
```

Comment:
DNS rebinding fix is for all OS not only fedora, centos or amzn

Reply:
OK. Have copied from old, and I would not change function here.